src/Components/Recommendation/Recommendation.spec.ct.js

```diff
@@ -56,6 +56,10 @@ describe('recommendation page for enabled recommendation with clusters enabled a
         </Intl>
       </MemoryRouter>
     );
+
+    cy.intercept('POST', '/api/insights-results-aggregator/v2/ack', {
+      statusCode: 204,
```

Comment:
where have you seen it? 201 should be when the recommendation is acked, not de-ack. I see a 301 (moved permanently) that I have no idea where is coming from. Should we investigate?

Reply:
Yes, there, I was commenting on the /ack one, and it returns 201. Also, it can be 200 if the rule was disabled already https://github.com/RedHatInsights/insights-results-smart-proxy/blob/master/server/api/v2/openapi.json#L367

![image](https://user-images.githubusercontent.com/31385370/177955034-e8152f61-0c66-445b-aa44-83b7a9c39e30.png)